rbac/providers/azure/aad_auth.py

```diff
@@ -35,10 +38,11 @@ def __init__(self):
 
     def get_token(self):
         """Get an access Token for Azure Active Directory Graph API."""
-        data = {'client_id': CLIENT_ID,
-                'scope': 'https://graph.microsoft.com/.default',
-                'client_secret': CLIENT_SECRET,
-                'grant_type': 'client_credentials'}
+        data = {'grant_type': GRANT_TYPE,
+                'client_id': CLIENT_ID,
+                'client_assertion_type': CLIENT_ASSERTION_TYPE,
```

Comment:
I believe client_assetion_type will break the other form of authentication.  Can you separate the cert token from the credential token get?